api/report.go

```diff
@@ -2,17 +2,34 @@ package api
 
 // Report contains the fields of a Preflight report
 type Report struct {
-	// Unique ID of the report
+	// Unique ID of the report.
 	ID string `json:"id"`
-	// Timestamp indicates when the report was generated
+	// PreflightVersion indicates the version of preflight this report was generated with.
+	PreflightVersion string `json:"preflight-version"`
+	// Timestamp indicates when the report was generated.
 	Timestamp Time `json:"timestamp"`
-	// Cluster indicates which was the target of the report
+	// Cluster indicates which was the target of the report.
 	Cluster string `json:"cluster"`
-	// Package indicates which package was used for the report
-	Package     string          `json:"package"`
-	Name        string          `json:"name"`
-	Description string          `json:"description,omitempty"`
-	Sections    []ReportSection `json:"sections,omitempty"`
+	// Package indicates which package was used for the report. (deprecated)
+	Package string `json:"package"`
+	// PackageInformation contains all the information about the package that was used to generate the report.
+	PackageInformation Package `json:"package-information"`
+	// Name is the name of the package that was used for this report.
+	Name string `json:"name"`
+	// Description is the description of the package that was used for this report.
+	Description string `json:"description,omitempty"`
+	// Sections contains the sections of the package that was used for this report.
+	Sections []ReportSection `json:"sections,omitempty"`
+}
+
+// Package contains all the details to identify a package.
+type Package struct {
```

Comment:
I was thinking we should also rename this to `PackageInformation`.

Reply:
ok, makes sense.